src/plugins/interactive-media/interactive-media-editor.js

```diff
@@ -207,11 +210,73 @@ function InteractiveMediaEditor({ content, onContentChanged }) {
 
   const handleChapterTitleChange = event => {
     const { value } = event.target;
-    const newChapters = [...chapters];
+    const newChapters = cloneDeep(chapters);
     newChapters[selectedChapterIndex] = { ...newChapters[selectedChapterIndex], title: value };
     changeContent({ chapters: newChapters });
   };
 
+  const handleChapterQuestionChange = event => {
+    const { value } = event.target;
+    const newChapters = cloneDeep(chapters);
+    newChapters[selectedChapterIndex] = { ...newChapters[selectedChapterIndex], question: value };
+    changeContent({ chapters: newChapters });
+  };
+
+  const handleChaperAnswerChanged = (index, event) => {
```

Comment:
Small typo here

Reply:
It was either a typo or the mood of the answer, it had a chaper feeling, whatever that would be like 😅 
Corrected, thanks!

Reply:
I was thinking along those lines too :laughing: